tests/test_router_bgp.rb

```diff
@@ -389,7 +440,9 @@ def test_routerbgp_default_graceful_restart
     assert_equal(300, bgp.default_graceful_restart_timers_stalepath_time,
                  "bgp graceful restart default timer value should be '300'")
     refute(bgp.default_graceful_restart_helper,
-           'graceful restart helper default value should be enabled = false')
+           'graceful restart helper default value ' \
+           'should be enabled = false') if
+      node.client.api == 'NXAPI'
```

Comment:
Ick, that's hard to parse. Should this be marked as a TODO or is this something that just plain doesn't work on XR?

```
if node.client.api == 'NXAPI'
  refute(bgp.default_graceful_restart_helper, ....
end
```